src/Notes/EditProductsOnTheMove.php

```diff
@@ -34,24 +36,25 @@ public static function get_note() {
 		}
 
 		// Check that the previous mobile app notes have not been actioned.
-		if ( WC_Admin_Notes_Mobile_App::has_note_been_actioned() ) {
+		if ( Mobile_App::has_note_been_actioned() ) {
 			return;
 		}
-
-		if ( WC_Admin_Notes_Real_Time_Order_Alerts::has_note_been_actioned() ) {
+		if ( Real_Time_Order_Alerts::has_note_been_actioned() ) {
```

Comment:
I'm getting all these related notes added at once.  I'm guessing because they're being added in a certain order.  Is this expected behavior or is this supposed to prevent them from showing? 

<img width="544" alt="Screen Shot 2020-10-05 at 11 11 08 AM" src="https://user-images.githubusercontent.com/10561050/95097478-342e2580-0736-11eb-8bb6-fdc315e8e34b.png">

Reply:
You would be getting them all in one go because your store is old enough to get all of the mobile notes. Since none of them are actioned they all get displayed. I think this is expected behaviour.

Reply:
(At least, nothing about the requirements that I've seen seem to say otherwise)